governance-proposal/TSC-Project-Lifecycle.md

```diff
@@ -0,0 +1,78 @@
+
+The TSC shall create, align and coordinate Projects that are ideally developed together, including possibly, the creation of sub-Projects. The TSC shall have the power to reorganize Projects and sub-Projects after sufficient review and discussion with the Projects and community involved.
+
+The TSC shall encourage new Projects and innovation in the technical community. New Projects enter the Node.js technical community through a Proposal to the TSC and if approved, are granted Incubation-state status.
+
+Projects shall change state following TSC reviews. Projects typically change states independently from each other, but can cooperate closely and leverage each other’s results. Projects graduate from Proposal-state, through Incubation-state and Mature-state to Core-state. Archived–state is a Project state reserved for those Projects no longer being actively developed or used by the community.
+
+The TSC shall have the authority to grandfather proposed Projects with a significant history and record to meet the project state descriptions below into a higher order state during the first 12 months after the first TSC meeting.
+
+| Project state |	Description |
+| ------------- | ----------- |
+| Proposal | Project does not formally exist yet, may not have real resources (yet), but is being worked on by the community to submit a formal proposal to the TSC. |
+| Incubation | Project has been approved by the TSC, has resources, but is recognized to be nascent. |
+| Mature | Project is fully functioning and stable, has achieved successful releases, has a Project Lead, but is not a required component of the platform.|
+| Core | Project is a required component of the Node.js platform.|
+| Archived | Project has been recognized as no longer being actively used or developed. This could be for a variety of reasons, e.g. project successfully accomplished its goals but is no longer used, project failed, etc., and has been archived as it's no longer a going concern.|
+ 
+### Project state transitions
+
+| From State | To State | Review |
+| ---------- | -------- | ------ |
+| `null` | Proposal | n/a |
+| Proposal | Incubation | Creation Review |
+| Incubation | Mature | Graduation Review |
+| Mature | Core | Core Review |
+| Proposal, Incubation, Mature, Core | Archived | Archive Review |
+
+## Reviews
+
+### Creation Review
+* Proposal posted for two weeks, evaluated on metrics of:
+ * Name is okay (e.g. no use of a trademark)
+ * Project contact name and email
+ * Description is complete
+ * Scope and project plan is well defined
+ * Resources are committed
+ * Initial Committers named
+ * Contributors have been identified
+ * Meets Foundation’s policies (e.g. IP Policy)
+ * Proposal has been socialized with potentially interested or affected existing Projects
+ * Proposal email has been sent to the TSC mailing list
+* Review by TSC: Confirm that the proposal is complete and the above listed requirements have been sufficiently met.
```

Comment:
The way I interpret this is only for projects that want to be under the umbrella of the Node.js Foundation. Nothing should stop anyone from creating a repo for any project wherever they would like, but that projects under the auspices of the Node.js project should have a bit more rigor around their setup.